packages/protocol/src/index.ts

```diff
@@ -27,13 +25,19 @@ const AirnodeRequesterRrpAuthorizerAddresses: { [chainId: number]: string } = {
 const mocks = {
   MockRrpRequesterFactory,
 };
-// TODO:
 const authorizers = {
   AirnodeRequesterRrpAuthorizerFactory,
 };
 
-// NOTE: For now, we only want to expose AirnodeRrp contract (and it's mock)
 export { AirnodeRrpAddresses, AirnodeRequesterRrpAuthorizerAddresses, AirnodeRrpFactory, mocks, authorizers };
-// Export the artifacts for API consumers using web3
-export { AirnodeRrpArtifact, MockRrpRequesterArtifact };
```

Comment:
We do not need to export the artifacts, because the ABI is available on the generated contracts from TypeChain...

I don't think end users will depend on protocol package - so I do not think we need to export these anymore.